guide/02-api-overview/release-notes.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>There are more notebooks, almost 20,.do we want to add all the names?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I'll update these today.

Reply:
@priyankatuteja @mohi9282 According to what I see merged into the public repo since the `1.8.4` release - there were 6 notebooks added (2 Guides and 4 Samples), with multiple fixes to various samples. I don't see that 20 new notebooks have been merged as of right now.